tensorflow/python/ops/image_ops_impl.py

```diff
@@ -849,6 +849,15 @@ def pad_to_bounding_box(image, offset_height, offset_width, target_height,
 
   This op does nothing if `offset_*` is zero and the image already has size
   `target_height` by `target_width`.
+  
+  Usage Example:
+    >>> x = [[[1., 2., 3.],
+    ...       [4., 5., 6.]],
+    ...       [[7., 8., 9.],
+    ...       [10., 11., 12.]]]
+    >>> padded_image = tf.image.pad_to_bounding_box(x, 1, 1, 4, 4)
+    >>> padded_image
+    <tf.Tensor: shape=(4, 4, 3), dtype=float32, numpy=array([...], dtype=float32)>
```

Comment:
We would prefer the numbers to be visible in the numpy array

Reply:
@boronhub said [...] syntax is better: https://github.com/tensorflow/tensorflow/pull/36056#discussion_r368853617. Should I add the numbers again?

Reply:
Yes please. If the numbers are deterministic they should show up. If they are not, the test should be rewritten to make them so. Only if the test cannot be rewritten in that way should you go for `...`

Reply:
All right.